### PR TITLE
refactor/flatlist: removed @shopify/flash-list, added react-native-gesture-handler as dependency

### PR DESCRIPTION
**Pull request by thetaungg**

### CODE REVIEW

File: src/components/DatePickerInline/YearPicker.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { useRef, useEffect, memo, useMemo } from 'react';
 import { StyleSheet, View } from 'react-native';
-import type { FlashList } from '@shopify/flash-list';
+import type { FlatList as GestureHandlerFlatList } from 'react-native-gesture-handler';
 import { useComponentStyles, useMolecules } from '../../hooks';
```
Comment (shashank1010) [inline src/components/DatePickerInline/YearPicker.tsx:4]: DatePickerInline has a file Swiper.native.tsx which renders a ScrollView that comes from react-native

Surprisingly, the View also in this file also comes from react-native

**Review by shashank1010 — CHANGES_REQUESTED** [review]

File: src/components/DatePickerInline/YearPicker.tsx (hunk 1 above)
Comment (thetaungg) [reply src/components/DatePickerInline/YearPicker.tsx:4]: :)
I must've overlooked.

Comment (thetaungg) [reply src/components/DatePickerInline/YearPicker.tsx:4]: Updated

File: stories/components/Accordion/Accordion.stories.tsx [hunk 2]
```diff
@@ -132,16 +173,18 @@ NestedAccordion.parameters = {
...
-            const { Accordion, AccordionItem, Icon, View, Text } = useMolecules();
+export const ExampleNestedAccordion = () => {
+    const { View, Text } = useMolecules();
+
     const leftElement = useCallback(
         ({ expanded, color }: AccordionHeaderElementProps) => (
             <Icon color={color} name={expanded ? 'chevron-up' : 'chevron-down'} size={24} />
         ),
-        [Icon],
+        [],
```
Comment (shashank1010) [inline stories/components/Accordion/Accordion.stories.tsx:183]: removed Icon?

**Review by shashank1010 — APPROVED** [review]

**Pull request merged**